apollo-runtime/src/test/java/com/apollographql/android/impl/CacheTest.java

```diff
@@ -355,6 +355,20 @@
     checkNoCachedResponse();
   }
 
+  @Test public void expireAfterRead() throws IOException {
+    enqueueResponse("src/test/graphql/allPlanetsResponse.json");
+    assertThat(apolloClient.newCall(new AllPlanets()).execute().isSuccessful()).isTrue();
+    checkCachedResponse("src/test/graphql/allPlanetsResponse.json");
+
+    assertThat(apolloClient.newCall(new AllPlanets()).expireAfterRead().execute().isSuccessful()).isTrue();
+    checkNoCachedResponse();
+    try {
+      apolloClient.newCall(new AllPlanets()).cache().execute();
+      fail("exception expected");
+    } catch (Exception expected) {
+    }
```

Comment:
Let's add 1 more  new call and execute to assert you get a new network response after expiring the last one.